contracts/contracts/core/vault/src/main.sw

```diff
@@ -1191,21 +1155,9 @@ fn _get_swap_fee_basis_points(
     asset_out: AssetId,
     rusd_amount: u256,
 ) -> u256 {
-    let is_stableswap = 
```

Comment:
Thes and the above functions for swapping can likely be completely removed, as single RLP asset means no swapping.

Reply:
done in the next PR [STAR-32]